translations/en.json

```diff
@@ -1757,5 +1757,17 @@
     "pending": "Pending",
     "negative": "Negative",
     "positive": "Positive",
+    "public": "Public",
```

Comment:
did you check if we already have some words ? i see we already have "Public" for example

Reply:
I got "Public" from rebasing develop, may be a result of some recent PR, I will check again